galsim/roman/__init__.py

```diff
@@ -25,21 +25,27 @@
 
 gain = 1.0
 pixel_scale = 0.11  # arcsec / pixel
-diameter = 2.37  # meters
+# Effective area now taken into account in the bandpass throughput (units of effective area, m^2) and sky background files.
+# To modify these assumptions, remove approximate factor of eff_area = 0.25 * np.pi * diameter**2 * (1. - obscuration**2) in meters^2 and rescale to new values of these parameters.
```

Comment:
This is a bit ambiguous now that the 'mean' collecting_area has been taken out again internally. Should clarify that nothing has changed for the user if these are used within galsim, only if someone uses the files directly?